### DISCUSSION TITLE
请求开发者能出一款能在安卓手表上使用的app

### DISCUSSION THREAD

**Discussion by cndoin** [gitbobobo/StreamMusic#673] [Ideas]
我非常希望开发者能改进一下app，应该为现在手表使用过程中安卓手机在播放时经常出现断链耳机，这个应该为手机蓝牙被手表占用经常耳机发不出声音，这个是能责怪谷歌安卓开发系统没有考虑周全，这个bug是无法解决的，所以我来请求开发者能出一款能在安卓手表上使用的应用程序，感谢每一位开发者！！！

**Comment by 17620139396** [comment]
Tv os什么时候能上线

**Reply by cndoin** [reply]
电视盒子的soc看一下 32 位还是 64 位一般 64 位的话安装安卓通用的版本一般来讲都可以用
